lib/Source/Tooling/Web/Perl.pm

```diff
@@ -37,14 +37,22 @@ sub critique ($env) {
 
 sub perldoc ($env) {
     my $r    = Plack::Request->new( $env );
+    my $name = substr $r->path, 1;
     my $args = $r->query_parameters;
+    my $func_name = !$args->{f};
 
-    return [ 400, [], [ 'You must specify either a function name' ]]
-        unless $args->{f};
+    my @cmd  = ($ENV{PERLDOC_BIN}, '-o', 'HTML');
```

Comment:
The PERLDOC_BIN variable is no longer accessible this way, you need to do `$env->{'source.tooling.env.PERLDOC_BIN'}`